extensions/firestore-sheets-sync/functions/src/index.ts

```diff
@@ -0,0 +1,134 @@
+import * as functions from 'firebase-functions';
+import { GaxiosError } from 'gaxios';
+import { google, sheets_v4 } from 'googleapis';
+
+import config from './config';
+
+const sheets = google.sheets('v4');
+const authClient = google.auth.getClient({
+  // keyFilename: 'extensions-testing-cae073db7903.json',
+  scopes: ['https://www.googleapis.com/auth/spreadsheets'],
+});
+
+const fields = config.fields.split(',');
```

Comment:
I think in other extensions we split in the config, so the config always returns an array,m e.g.:

https://github.com/firebase/extensions/blob/next/firestore-translate-text/functions/src/config.ts#L19 (note the set also to remove duplicate field names)